Sources/Lexer/Token.swift

```diff
@@ -19,7 +19,7 @@ public struct Token: Equatable, SourceEntity, CustomStringConvertible {
   // MARK: - Kind
   public enum Kind: Equatable, CustomStringConvertible {
     // Newline
-    case newline
+    case `newline`
```

Comment:
Why did you backtick-escape this? `newline` is not an identifier in Swift (as far as I know)